learn/getting_started/quick_start.md

```diff
@@ -269,6 +269,10 @@ Meilisearch offers a search preview where you can preview search results. It com
 
 ![Meilisearch's search preview showing the movies index](/search_preview/default.png)
 
+::: note
+For security reasons, the search preview is only available in [development mode.](/learn/configuration/instance_options.md#environment)
+:::
```

Comment:
These custom containers (note, tip, warning etc.) are mainly useful for making text stand out from their surroundings. In this case there's no surrounding text, so I personally don't think it's necessary.

Other than that, it's a nice addition! LGTM.

```suggestion
For security reasons, the search preview is only available in [development mode.](/learn/configuration/instance_options.md#environment)
```